com.discordapp.Discord.appdata.xml

```diff
@@ -20,6 +20,8 @@
       <li>Available on Linux, macOS, Windows, iOS, Android, and your web browser.</li>
     </ul>
     <p>Discord is written in Javascript, React, Elixir, and Rust. The desktop application uses the electron framework.</p>
+    <p>The flatpak version runs in a sandbox to provide better safety and privacy for users. However, this sandboxing prevents the following features from working: Game Activity, Unrestricted File Access, Rich Presence. Check the README in the Github repo for details.</p>
```

Comment:
I would say that it prevents "the following features from working out of the box" instead because otherwise it might imply that this can't be fixed by people using the application.

Reply:
```suggestion
    <p>The flatpak version runs in a sandbox to provide better safety and privacy for users. However, this sandboxing prevents the following features from working out of the box: Game Activity, Unrestricted File Access, Rich Presence. Check the README in the Github repo for details.</p>
```